### PR TITLE
TE-406 Table text alignment

### PR DESCRIPTION
**Pull request by Jonnygwi**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-406)

### What **one** thing does this PR do?
Aligns all table cells in the first column to the left and all others to the center

### Any other notes
image

### CODE REVIEW

File: src/components/collections/Table/component.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ export const Component = ({ tableHeadings, tableBody, tableId }) => (
         <Table.Row key={buildKeyFromStrings(tableId, `row${rowIndex}`)}>
           {row.map((cell, cellIndex) => (
             <Table.Cell
+              textAlign={cellIndex === 0 ? 'left' : 'center'}
```
Comment (marcdavi-es) [inline src/components/collections/Table/component.js]: DRY this up by isolating as a util

`export const getTextAlign = index => index === 0 ? 'left' : 'center'`

File: src/components/collections/Table/utils/getTextAlign.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { getTextAlign } from './getTextAlign';
+
+describe('getTextAlign', () => {
+  const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
```
Comment (marcdavi-es) [inline src/components/collections/Table/utils/getTextAlign.spec.js]: Do you think we should expand this to all numbers?

Comment (marcdavi-es) [reply src/components/collections/Table/utils/getTextAlign.spec.js]: 🚨JOKE ALERT 🚨

File: src/components/collections/Table/component.js [hunk 3]
```diff
@@ -26,6 +29,7 @@ export const Component = ({ tableHeadings, tableBody, tableId }) => (
         <Table.Row key={buildKeyFromStrings(tableId, `row${rowIndex}`)}>
           {row.map((cell, cellIndex) => (
             <Table.Cell
+              textAlign={getTextAlign(cellIndex)}
```
Comment (davidpelayo) [inline src/components/collections/Table/component.js]: It feels to me that the table component needs hovering/focus effects on rows and cells to navigate the content properly

File: src/components/collections/Table/utils/getTextAlign.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/**
+ * @param  {Number} index
+ * @return {String}
+ */
+export const getTextAlign = index => (index === 0 ? 'left' : 'center');
```
Comment (davidpelayo) [inline src/components/collections/Table/utils/getTextAlign.js]: I would expect by this util to be able to give me any position, whereas is just a two positions conditioner. I think renaming is needed here, although it's just a detail

File: src/components/collections/Table/component.js (hunk 3 above)
Comment (jams676) [reply src/components/collections/Table/component.js]: The table is not interactive at all though. There's a small off-white/gray color change between rows that should be enough to differentiate from rows. Let me know if you have any other suggestions examples, but I think hover is unnecessary

**Pull request merged**